<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skyrim Mod Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Images/Original.ico</normaloff>:/Images/Original.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="modTab">
       <attribute name="title">
        <string>Mods</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTreeView" name="modsView">
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
          <widget class="QTreeView" name="treeView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pluginTab">
       <attribute name="title">
        <string>Plugins</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QTreeView" name="pluginsView">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="rightSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bossButton">
            <property name="text">
             <string>Launch BOSS</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="leftSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSkyrim">
    <property name="title">
     <string>Skyrim</string>
    </property>
    <addaction name="actionLaunch"/>
    <addaction name="actionLaunchSkse"/>
    <addaction name="separator"/>
    <addaction name="actionOpenGameFolder"/>
    <addaction name="actionOpenModsFolder"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuCheck_for_updates">
     <property name="title">
      <string>Check for mod updates</string>
     </property>
     <addaction name="actionOnce_a_day"/>
     <addaction name="actionOn_app_launch"/>
     <addaction name="separator"/>
     <addaction name="actionCheck_now"/>
    </widget>
    <addaction name="actionSuMode"/>
    <addaction name="actionChangeSkyrim"/>
    <addaction name="actionChangeMods"/>
    <addaction name="actionChangeBoss"/>
    <addaction name="separator"/>
    <addaction name="menuCheck_for_updates"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionReadme"/>
   </widget>
   <addaction name="menuSkyrim"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLaunch">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Original.ico</normaloff>:/Images/Original.ico</iconset>
   </property>
   <property name="text">
    <string>Launch</string>
   </property>
  </action>
  <action name="actionLaunchSkse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/Original.ico</normaloff>:/Images/Original.ico</iconset>
   </property>
   <property name="text">
    <string>Launch SKSE</string>
   </property>
  </action>
  <action name="actionOpenGameFolder">
   <property name="text">
    <string>Open game folder</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
  <action name="actionSuMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced mode</string>
   </property>
  </action>
  <action name="actionChangeSkyrim">
   <property name="text">
    <string>Change Skyrim folder</string>
   </property>
  </action>
  <action name="actionChangeMods">
   <property name="text">
    <string>Change mods folder</string>
   </property>
  </action>
  <action name="actionChangeBoss">
   <property name="text">
    <string>Change BOSS folder</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenModsFolder">
   <property name="text">
    <string>Open mods folder</string>
   </property>
  </action>
  <action name="actionInstallSelection">
   <property name="text">
    <string>Install selection</string>
   </property>
  </action>
  <action name="actionUninstallSelection">
   <property name="text">
    <string>Uninstall selection</string>
   </property>
  </action>
  <action name="actionOnce_a_day">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Once a day</string>
   </property>
  </action>
  <action name="actionOn_app_launch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On app launch</string>
   </property>
  </action>
  <action name="actionCheck_now">
   <property name="text">
    <string>Check now</string>
   </property>
  </action>
  <action name="actionCheckSelection">
   <property name="text">
    <string>Check selection</string>
   </property>
  </action>
  <action name="actionUncheckSelectio">
   <property name="text">
    <string>Uncheck selection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
